.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "python.autoComplete.extraPaths": [
+        "./tests",
+    ],
+    "editor.formatOnSave": true,
+    "editor.formatOnSaveTimeout": 1000,
+    "python.formatting.provider": "black",
+    "python.linting.flake8Args": [
+        "--max-line-length=88"
```

Comment:
Can we use here the alternative warning (something called bear or similar) instead of the strict check? This will fail if not sometimes

Reply:
Good point, solved.